src/plugins/lua/mx_check.lua

```diff
@@ -305,6 +311,14 @@ if opts then
     one_shot = true,
     one_param = true,
   })
+  rspamd_config:set_metric_symbol({
+    name = settings.symbol_white_mx,
+    score = -0.00,
```

Comment:
Sign and precision do not make sense with zero score. I'd suggest to change the score to `0.0`.